### ISSUE TITLE
Tracking: import test cases from other DBs

### ISSUE THREAD

**Issue by lmatz** [risingwavelabs/risingwave#5156]
Will fill in what we have now, and what we may import.

**Comment by lmatz** [comment]
I agree that postgres contains too many features that we will never need

but the test cases from `duckdb` and `sqllogictest` seem relatively clean, i.e. a limited set of feature but test many combinations of them.

I suggest after reviewing `duckdb`, we proceed to `sqllogictest` next.

**Comment by xiangjinwu** [comment]
Also note that duckdb and sqlite is not PostgreSQL-compatible, and may contain features not supported by PostgreSQL. It is not a goal to able to pass their tests unmodified.
